tests/py/test_billing_payday.py

```diff
@@ -29,8 +29,91 @@ def test_payday_moves_money(self):
         obama = Participant.from_username('obama')
         picard = Participant.from_username('picard')
 
-        assert picard.balance == D('6.00')
-        assert obama.balance == D('3.41')
+        assert picard.balance == D(MINIMUM_CHARGE)
+        assert obama.balance == D('0.00')
+        assert obama.get_due('TheEnterprise') == D('0.00')
+
+    @mock.patch.object(Payday, 'fetch_card_holds')
+    def test_payday_moves_money_cumulative_above_min_charge(self, fch):
+        Enterprise = self.make_team(is_approved=True)
+        self.obama.set_payment_instruction(Enterprise, '5.00')  # < MINIMUM_CHARGE
+        # simulate already due amount
+        self.db.run("""
+
+            UPDATE payment_instructions ppi
+               SET due = '5.00'
+             WHERE ppi.participant = 'obama'
+               AND ppi.team = 'TheEnterprise'
+
+        """)
+
+        fch.return_value = {}
+        Payday.start().run()
+
+        obama = Participant.from_username('obama')
+        picard = Participant.from_username('picard')
+
+        assert picard.balance == D('10.00')
+        assert obama.balance == D('0.00')
+        assert obama.get_due('TheEnterprise') == D('0.00')
+
+    @mock.patch.object(Payday, 'fetch_card_holds')
+    def test_payday_preserves_due_until_charged(self, fch):
+        Enterprise = self.make_team(is_approved=True)
+        self.obama.set_payment_instruction(Enterprise, '2.00')  # < MINIMUM_CHARGE
+
+        fch.return_value = {}
+        Payday.start().run()    # payday 0
+
+        assert self.obama.get_due('TheEnterprise') == D('2.00')
+
+        self.obama.set_payment_instruction(Enterprise, '3.00')  # < MINIMUM_CHARGE
+        self.obama.set_payment_instruction(Enterprise, '2.50')  # cumulatively still < MINIMUM_CHARGE
```

Comment:
Cumulative with what? Not cumulative with the 3.00 just above, since this replaces that, right?